### PR TITLE
Reorganise chunking, review of dask_intro

### PR DESCRIPTION
**Pull request by tinaok**

### CODE REVIEW

File: tutorial/pangeo101/chunking_introduction.ipynb [hunk 1]
```diff
@@ -194,7 +194,776 @@
      "name": "#%%\n"
```
Comment (pl-marasco) [inline tutorial/pangeo101/chunking_introduction.ipynb:194]: <p>Best option would be to take the first 22 files to let users appreciate how the vegetation has changed during the year but I agree that we have to make as quick as possible. What about taking the last month [18:23]? </p><p> </p><p>in other words files:</p><p> </p><p> 'foss4g-data/CGLS_LTS_1999_2019/c_gls_NDVI-LTS_1999-2019-0701_GLOBE_VGT-PROBAV_V3.0.1.nc', </p><p> 'foss4g-data/CGLS_LTS_1999_2019/c_gls_NDVI-LTS_1999-2019-0711_GLOBE_VGT-PROBAV_V3.0.1.nc', </p><p> 'foss4g-data/CGLS_LTS_1999_2019/c_gls_NDVI-LTS_1999-2019-0721_GLOBE_VGT-PROBAV_V3.0.1.nc', </p><p> 'foss4g-data/CGLS_LTS_1999_2019/c_gls_NDVI-LTS_1999-2019-0801_GLOBE_VGT-PROBAV_V3.0.1.nc', </p><p> 'foss4g-data/CGLS_LTS_1999_2019/c_gls_NDVI-LTS_1999-2019-0811_GLOBE_VGT-PROBAV_V3.0.1.nc',</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (pl-marasco) [inline tutorial/pangeo101/chunking_introduction.ipynb:194]: <p>Agree, here would only makes confusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Comment (tinaok) [reply tutorial/pangeo101/chunking_introduction.ipynb:194]: s3path = 's3://foss4g-data/CGLS_LTS_1999_2019/c_gls_NDVI-LTS_1999-2019-0[7-8]*.nc'
suits you? I'm testing on https://pangeo-foss4g.vm.fedcloud.eu/jupyterhub/ and with that it is 5 times faster than Pangeo US configuration, with 6 files, it takes 10 seconds, so I think it is acceptable.

Comment (pl-marasco) [reply tutorial/pangeo101/chunking_introduction.ipynb:194]: <p>Fine by me ! </p>

**Comment by annefou** [comment]
@tinaok can you clean the outputs of the notebooks before we merge? thanks.

**Comment by tinaok** [comment]
Hi, I'm reviewing dask_introduction.ipynb, does anyone know any reference for 'It is interesting to note that at first, Dask has been created to handle data that is larger than memory, on a single computer. It then was extended with Distributed to compute data in parallel over cluster of computers.'

**Comment by guillaumeeb** [comment]
> does anyone know any reference for 'It is interesting to note that at first, Dask has been created to handle data that is larger than memory, on a single computer. It then was extended with Distributed to compute data in parallel over cluster of computers.'

@tinaok, see https://coiled.io/blog/history-dask/

**Pull request merged**